integration-tests/src/test/java/oracle/weblogic/kubernetes/ItCrossDomainTransaction.java

```diff
@@ -232,8 +233,12 @@ private static void addToPropertyFile(String propFileName, String domainNamespac
   private static void buildApplicationsAndDomains() {
 
     //build application archive
+    // Path targetDir = Paths.get(WORK_DIR, this.getClass().getSimpleName());
+    // non-static variable this cannot be referenced from a static context
+
+    Path targetDir = Paths.get(WORK_DIR, "ItCrossDomainTransaction/txforward");
```

Comment:
Use ItCrossDomainTransaction.class.getName() to get the class name or other ways which works for refactoring

Reply:
I was trying to use this.getClass().getSimpleName() which gave me the compilation error as follows ...
non-static variable this cannot be referenced from a static context

So I moved to hardcode the class name as String. Your suggestion is also a hard-coded solution.

Reply:
Modified
    ItCrossDomainTransaction/txforward"-->  ItCrossDomainTransaction.class.getName() + "/txforward"

Reply:
Not sure how this is different from what you had before- except make an additional call.

Reply:
As per Vanaja this will help to refactor the code using IDE in future.